nzgoal/NZGOAL-Version-2.md

```diff
@@ -143,35 +139,29 @@ NZGOAL Review and Release Process Decision Trees	32
 
 Appendix –	Recommended Copyright and Licensing Statements, and Attribution Statements	35
 
-# 
-
-# 
-
-# 
-
-# Introduction
+## Introduction
 
 ### Purpose
 
-State Services agencies generate vast quantities of copyright and non-copyright material. It is widely recognised, in New Zealand and abroad, that significant creative and economic potential may lie dormant in such material when locked up in agencies and not released on terms allowing re-use by others. That potential is two-fold:
+ 1. State Services agencies generate vast quantities of copyright and non-copyright material. It is widely recognised, in New Zealand and abroad, that significant creative and economic potential may lie dormant in such material when locked up in agencies and not released on terms allowing re-use by others. That potential is two-fold:
 
-individuals, non-profit and commercial organisations can leverage this material for creative, cultural and economic growth, improved environmental sustainability, greater productivity, and the wider public benefit; and
+    (a) individuals, non-profit and commercial organisations can leverage this material for creative, cultural and economic growth, improved environmental sustainability, greater productivity, and the wider public benefit; and
 
-experts and others can contribute to improved policy development and more efficient financial performance by government through being able to access, manipulate and provide feedback on such material. 
+    (b) experts and others can contribute to improved policy development and more efficient financial performance by government through being able to access, manipulate and provide feedback on such material. 
 
-NZGOAL provides a means of realising this potential. It:
+ 2. NZGOAL provides a means of realising this potential. It:
 
-sets out a series of open licensing and open access principles, for copyright works and non-copyright material respectively;
+    (a) sets out a series of open licensing and open access principles, for copyright works and non-copyright material respectively;
 
-advocates the use of:	
+    (b) advocates the use of:	
 
-Creative Commons licences for those State Services agencies’ copyright works which are appropriate for release and re-use; and
+        (i) Creative Commons and Open Source licences for those State Services agencies’ copyright works which are appropriate for release and re-use; and
```

Comment:
I have simply added "and Open Source" to this section as the rest of the wording is completely valid for OSS, Open Data and Content. OSS is simply not covered by CC licensing and later in the framework these licenses such as BSD, GPL, Apache would be detailed and explored.